data/meson.build

```diff
@@ -19,7 +19,7 @@ appstream_file = i18n.merge_file(
   output: 'dev.tchx84.Portfolio.appdata.xml',
   po_dir: '../po',
   install: true,
-  install_dir: join_paths(get_option('datadir'), 'appdata')
+  install_dir: join_paths(get_option('datadir'), 'metainfo')
```

Comment:
Since we are doing this, would you mind renaming [1] the XML file as well ? Plus, make sure the current XML is up to the latest standards.

[1] `s/appdata.xml/metainfo.xml`

Reply:
Thanks, done. The appstream XML does not yeld any warning (but I am no expert) eithier during build or later on using dedicated validation tool as lintian, so I consider it valid as is.